test/build/error-config-invalid/error-config-invalid.test.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const {setupBuildTest, readFiles} = require('../../test-utils');
+
+const cwd = path.join(__dirname, 'build');
+
+let files = {};
+let error;
+
+describe('install not being an array', () => {
+  beforeAll(() => {
+    try {
+      setupBuildTest(__dirname);
+    } catch(err) {
+      error = err;
```

Comment:
👍🏻  Makes sense. I should probably use this pattern rather than `.toThrowError()` to take advantage of partial string matching

Reply:
this was more about me not being familiar enough with jest to think of doing it that way... but yeah searching for the error message seemed useful here too.